docs/install-faq.rst

```diff
@@ -1,7 +1,6 @@
 * I cannot install the package and it reports the error "XXX is not a supported wheel on this platform".
 
-   One possibility is that you are using an older version of pip. Try to upgrade your pip to a version later than "19.0.0",
-e.g., use the following command:
+   One possibility is that you are using an older version of pip. Try to upgrade your pip to a version later than "19.0.0", e.g., use the following command:
 
```

Comment:
Maybe this PR should clarify in Install-FAQ that: 

“Mac users need to install the linked custom MXNet wheel in order to use TextPrediction. To just use TabularPrediction, Mac users can either install the custom MXNet wheel or simply install MXNet via: `pip install mxnet`